README.md

```diff
@@ -79,12 +79,12 @@ The first 64bytes of a CacheAdvance is reserved for storing metadata about the f
 
 ## Requirements
 
-* Xcode 11.2 or later.
-* iOS 12 or later.
-* tvOS 12 or later.
-* watchOS 5 or later.
-* macOS 10.14 or later.
-* Swift 5.0 or later.
+* Xcode 12.4 or later.
```

Comment:
I'm somewhat tempted to go straight to Swift 5.9 and ignore older Xcode versions, but continuing to support older Xcode versions enables a test suite across more Apple platform versions, and currently there are zero code changes required to support Swift/Xcode versions above Swift 5.5.